pyecma376_2/package_model.py

```diff
@@ -645,7 +642,7 @@ def normalize_part_name(part_name: str) -> str:
     return part_name.lower()
 
 
-RE_PART_NAME = re.compile(r'^(/[A-Za-z0-9\-\._~%:@!$&\'()*+,;=]*[A-Za-z0-9\-_~%:@!$&\'()*+,;=])+$')
+RE_PART_NAME = re.compile(r'^(/[A-Za-z0-9\-\._~%:@!$&\'()*+,;= ]*[A-Za-z0-9\-_~%:@!$&\'()*+,;= ])+$')
```

Comment:
Can you quickly confirm that this is the only location where you changed the logic of the code?

Reply:
Yes